harness/determined/core/_train.py

```diff
@@ -221,6 +282,16 @@ def report_validation_metrics(self, steps_completed: int, metrics: Dict[str, Any
             f"report_validation_metrics(steps_completed={steps_completed} metrics={metrics})"
         )
 
+    def _auto_upload_tb_files(self) -> None:
+        logger.info("_auto_upload_tb_files")
+
+    def upload_tb_files(
+        self,
+        selector: Callable[[pathlib.Path], bool] = lambda _: True,
+        mangler: Callable[[pathlib.Path, int], pathlib.Path] = lambda p, __: p,
+    ) -> None:
+        logger.info("upload_tb_files()")
+
```

Comment:
extraneous edit, it looks like

Reply:
Not sure what you mean. I have to explicitly override `upload_tb_files` because the parent version refers to `self._tb_mgr` which is not defined in `DummyTrainContext`.

Reply:
oh, I'm a dummy (lol)